portal-web/docroot/html/taglib/ui/page_iterator/lexicon/start.jsp

```diff
@@ -46,6 +46,17 @@ if (end > total) {
 String deltaURL = HttpUtil.removeParameter(url, namespace + deltaParam);
 
 NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
+
+if (forcePost && (portletURL != null) && Validator.isNotNull(url)) {
```

Comment:
@inacionery

1. Is portletURL.toString() the same as url?
2. After I rebased with master, I'm having problems starting up portal.